frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html

```diff
@@ -141,6 +144,7 @@
                                 </div><div class="bundle-offering-a__print-option__price">
                                     @bundleVariant.prettyMonthlyPrice(SatGW)
                                 </div>
+                                <div class="bundle-offering-b__print-option__id">saturday-guardian-weekly</div>
```

Comment:
I think you should call this 'satgwe' to make it a unique search string. Right now 'saturday' is substring of this and a google analytics filter may bring back both